src/main/java/hudson/plugins/jira/JiraSession.java

```diff
@@ -66,6 +73,24 @@ public void addComment(String issueId, String comment,
         service.addComment(issueId, comment, groupVisibility, roleVisibility);
     }
 
+    public void addLabels(String issueId, List<String> labels) {
```

Comment:
Ok, i add something like this

/**
     * Adds new labels to the existing issue.
     * Old labels remains untouched.
     *
     * @param issueId Jira issue ID like "MNG-1235".
     * @param labels New labels to add.
     */